fundingmanager.go

```diff
@@ -1188,8 +1228,10 @@ func (f *fundingManager) handleFundingSigned(fmsg *fundingSignedMsg) {
 		},
 	}
 
+	f.wg.Add(1)
```

Comment:
Nice, this had to be added to make sure the goroutine was properly shutdown between tests.

Reply:
💯